orchagent/neighorch.cpp

```diff
@@ -1199,6 +1201,434 @@ bool NeighOrch::removeNeighbor(const NeighborEntry &neighborEntry, bool disable)
     return true;
 }
 
+/**
+ * @brief Creates a neighbor add entry and adds it to bulker.
+ * @param ctx NeighborBulkContext contains neighbor information and list of object statuses.
+ */
+bool NeighOrch::addBulkNeighbor(NeighborBulkContext& ctx)
```

Comment:
does it use bulk neighbor only for mux scenario or even for the existing path? I think we should enable it only for mux flow.

Reply:
it's only for mux flow for this PR